scio-hdfs/src/test/scala/com/spotify/scio/hdfs/HdfsTapTest.scala

```diff
@@ -79,6 +114,32 @@ class HdfsTapTest extends PipelineSpec {
     FileUtils.deleteDirectory(dir)
   }
 
+  it should "support default compression with saveAsHdfsTextFile" in {
```

Comment:
Do you also want a `it should "support user conf with saveAsHdfsTextFile"` test?